plugins/coverimage.koplugin/main.lua

```diff
@@ -36,10 +38,12 @@ local CoverImage = WidgetContainer:new{
 
 function CoverImage:init()
     self.cover_image_path = G_reader_settings:readSetting("cover_image_path") or "cover.png"
+    self.cover_image_background = G_reader_settings:readSetting("cover_image_background") or "none"
```

Comment:
It should probably be enabled by default. This shouldn't take too much time and it would ensure it works out of the box on all devices.

Reply:
OK, set it to black as this is the fastest method and it fits best to my Tolino